alexandria/core/serializers.py

```diff
@@ -206,26 +208,34 @@ def validate(self, *args, **kwargs):
         `validator_classes` attribute of the FileViewSet.
         """
         validated_data = super().validate(*args, **kwargs)
-        if validated_data.get(
-            "variant"
-        ) != models.File.Variant.ORIGINAL and not validated_data.get("original"):
-            file_variant = validated_data.get("variant")
-            raise ValidationError(
-                f'"original" must be set for variant "{file_variant}".'
-            )
-
-        if (
-            variant := validated_data.get("variant")
-        ) == models.File.Variant.ORIGINAL and validated_data.get("original"):
+        variant = validated_data.get("variant")
+        if variant != models.File.Variant.ORIGINAL and not validated_data.get(
+            "original"
+        ):
+            raise ValidationError(_(f'"original" must be set for variant "{variant}".'))
```

Comment:
An `f` string inside gettext is not gonna work. You can do that this way:
```suggestion
            raise ValidationError(_('"original" must be set for variant "%(variant)s".') % {"variant": variant})
```

Reply:
didnt know, makes sense. Fixes